src/main.ts

```diff
@@ -71,6 +71,7 @@ async function start () {
 
 // execution and error handling
 start().catch(error => {
+  Sentry.captureException(error)
```

Comment:
I'd love to see a comment here indicating the context for why to log this error (but not others).
PS I love comments that help me understand what/why the code is doing something :)